src/components/customHead.js

```diff
@@ -13,6 +13,7 @@ const CustomHead = () => (
         />
         <link href="/static/styles/main.css" rel="stylesheet" />
         <link href="/static/styles/puzzle.css" rel="stylesheet" />
+        <link href="/static/styles/responsive.css" rel="stylesheet" />
         <link
             rel="stylesheet"
             href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css"
```

Comment:
I hadn't noticed this one, but this breaks offline usage, no?

Reply:
This is normally cached too. I'll check that tomorrow.